packages/examples/src/molecules/water.substance

```diff
@@ -0,0 +1,18 @@
+Hydrogen h
+Oxygen o1, o2
+Bond b1 := MakeSingleBond(h, o1)
+Bond b2 := MakeSingleBond(h, o2)
+-- Electron o11 := ValenceElectron(o1)
+-- Electron o12 := ValenceElectron(o1)
+-- Electron o13 := ValenceElectron(o1)
+-- Electron o14 := ValenceElectron(o1)
+-- Electron o15 := ValenceElectron(o1)
+-- Electron o16 := ValenceElectron(o1)
```

Comment:
Is there a reason for leaving these `ValenceElectron`s commented out in the Substance program?

Reply:
good catch! fixed